.github/ISSUE_TEMPLATE/bug-reporting.yml

```
@@ -0,0 +1,36 @@
+name: Bug reporting
+description: Reporting new bugs
+title: 'BUG -'
+labels: bug
+projects: IMAP
+body:
+- type: textarea
+  attributes:
+    label: "## Description of the issue"
+- type: textarea
+  attributes:
+    label: "### Steps to reproduce the issue
```

```suggestion
    label: "Steps to reproduce the issue
```

In the yaml the "###" gets rendered with the title, so we should remove all of the "#"'s from these files.